utils/ogmios-client/tests/query_ledger_state.rs

```diff
@@ -0,0 +1,123 @@
+#![cfg(feature = "jsonrpsee-client")]
+
+use jsonrpsee::{http_client::HttpClient, server::Server, RpcModule};
+use ogmios_client::{
+	query_ledger_state::{EpochBoundary, EpochParameters, EraSummary, QueryLedgerState},
+	types::{SlotLength, TimeSeconds},
+};
+use serde_json::json;
+use std::net::SocketAddr;
+use tokio::sync::OnceCell;
+
+const SERVER_ADDRESS: OnceCell<SocketAddr> = OnceCell::const_new();
```

Comment:
This `OnceCell` here is more than required for a single test. IDK if it should stay as something usable when more tests are implemented?

Reply:
How expensive is creating the server for each test? Imo it's better to have the mocked values defined together with each unit test, than to have one central definition that handles all cases.

Reply:
It is cheap. I've refactored to have a handler definition inside test.